test/benchmarks/benchmark_adler32.cc

```
@@ -94,6 +94,10 @@ BENCHMARK_ADLER32(neon, adler32_neon, test_cpu_features.arm.has_neon);
 BENCHMARK_ADLER32(neon_dotprod, adler32_neon_dotprod, test_cpu_features.arm.has_neon && test_cpu_features.arm.has_dotprod);
 #endif
 
+#ifdef MIPS_MSA
+BENCHMARK_ADLER32(msa, adler32_msa, test_cpu_features.mips.has_msa);
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If BENCHMARK_DEFINE_F is a macro then please configure it.

(unknownMacro)